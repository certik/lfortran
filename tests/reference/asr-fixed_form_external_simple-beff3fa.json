{
    "basename": "asr-fixed_form_external_simple-beff3fa",
    "cmd": "lfortran --fixed-form --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/fixed_form_external_simple.f",
    "infile_hash": "5fb4d1f0c148a2dcac928b0d1bde13987cd8b687962a9f98b2459ac9",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-fixed_form_external_simple-beff3fa.stdout",
    "stdout_hash": "f46fd638111c6dee26c0db7c31add4ffbabae4e0367df90fbacb0cd5",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}