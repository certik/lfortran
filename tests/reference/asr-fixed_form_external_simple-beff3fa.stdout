(TranslationUnit (SymbolTable 1 {inner_scope: (Function (SymbolTable 4 {}) inner_scope [] [] [] () Source Public Interface () .false. .false. .false.), main: (Program (SymbolTable 3 {b: (Variable 3 b Local () () Default (Integer 4 []) Source Public Required .false.), c: (Variable 3 c Local () () Default (Integer 4 []) Source Public Required .false.)}) main [] [(SubroutineCall 1 inner_scope () [((Var 3 b)) ((Var 3 c))] ())]), outer_scope: (Function (SymbolTable 2 {a: (Variable 2 a Unspecified () () Default (Integer 4 []) Source Public Required .false.)}) outer_scope [(Var 2 a)] [] [(Print () [(Var 2 a)] () ())] () Source Public Implementation () .false. .false. .false.)}) [])
